yesod-auth/Yesod/Auth/Email.hs

```diff
@@ -549,7 +580,7 @@ registerHelper allowUsername forgotPassword dest = do
                     _ -> Nothing
 
     case eidentifier of
-      Left route -> loginErrorMessageI dest route
+      Left failMsg -> loginErrorMessageI dest failMsg
```

Comment:
This is just a renaming of the `route` variable to `failMsg` in the local scope of this case statement, since the value is in fact a `Msg` and not an `AuthRoute` as the old name implies.

Reply:
Thanks!